content/docs/concepts/clusters.mdx

```diff
@@ -0,0 +1,205 @@
+---
+id: clusters
+title: Clusters
+description: This document describes Clusters in Pomerium Zero.
+keywords: [clusters, pomerium zero]
+sidebar_label: Clusters
+sidebar_position: 1
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Clusters in Pomerium Zero
+
+This document describes Clusters in Pomerium Zero.
+
+## Overview
+
+A **Cluster** represents a single Pomerium deployment managed by Pomerium Zero, our hosted control plane. You can think of a Pomerium cluster as a completely distinct deployment of Pomerium that allows you to effectively shard your deployments for performance, governance, security, or management purposes.
+
+When you deploy a cluster, it gets its own:
+
+- **Cluster identity**
+- **Cluster domain**
+- **Cluster settings**
+- **Routes and Policies**
+- **Wildcard certificates**
+
+### Cluster use cases: Why clusters?
+
+#### All-in-one mode deployment model
+
+Pomerium consists of 4 services:
+
+- Proxy service
+- Authenticate service
+- Authorize service
+- Databroker service
+
+A cluster deploys these services in all-in-one mode, meaning the configuration for each service is managed from a single cluster instance. All inter-component communications are internally connected, with the exception of the Databroker service, which is an external dependency.
+
+#### Multiple cluster instances
+
+You can deploy multiple clusters from a single cluster identity (see [**Cluster identity**](#cluster-identity) below for more information). The ability to deploy multiple cluster instances means you can manage and run completely separate configurations for workloads that may otherwise be incompatible in a single cluster instance.
+
+### Cluster identity
+
+When you create an account in Pomerium Zero, a new **Cluster Token** is generated and assigned to your personal account. This cluster token, which is provided to you during onboarding, is required to deploy your cluster instance.
+
+You can generate a new cluster token at any time. See the [**Rotate cluster identity**](#rotate-cluster-identity) steps below for more information.
+
+#### Rotate cluster identity
+
+A cluster token does not have a set expiration time. However, you may want to rotate the cluster identity in the Zero Console for security reasons. For example:
+
+- A malicious insider could use the token to view the cluster's configuration, which may contain sensitive data
+- You may lose track of the existing token, and want a new one
+
+:::caution
+
+Rotating the cluster identity will invalidate the existing cluster token.
+
+:::
+
+To rotate your cluster identity in the Zero Console:
+
+1. From the navigation bar dropdown, select **Manage Clusters**
+1. Under **Actions**, select the **edit** icon
+1. Select the **Domains** tab
+1. In the **Cluster Identity** field, select **Rotate Identity**
```

Comment:
the described UI does not exist. it is currently in the _Manage > Settings > Advanced_ and is called **Rotate Token*

<img width="1019" alt="image" src="https://github.com/pomerium/documentation/assets/673131/effba922-f26b-4243-ba1c-be3fd02bcf26">